store/tikv/2pc.go

```diff
@@ -652,6 +652,10 @@ func (c *twoPhaseCommitter) doActionOnGroupMutations(bo *Backoffer, action twoPh
 		// by test suites.
 		secondaryBo := NewBackofferWithVars(context.Background(), CommitMaxBackoff, c.txn.vars)
 		go func() {
+			failpoint.Inject("skipCommitSecondaryKeys", func() {
+				failpoint.Return()
+			})
```

Comment:
How about to make this failpoint more general (eg. like the following) so that we can either disable it by `return("skip")` or delay it by `sleep(1000)`?
```go
failpoint.Inject("beforeCommitSecondaries", func(v failpoint.Value) {
	if s, ok := v.(string); !ok {
	} else if s == "skip" {
		failpoint.Return()
	}
})
```